deploy/kubernetes/operator.nsx.vmware.com_v1_ncpinstall_cr.yaml

```diff
@@ -5,3 +5,4 @@ metadata:
   namespace: nsx-system-operator
 spec:
   ncpReplicas: 1
+  disableTag: true
```

Comment:
How about changing disableTag to addNodeTag?

Reply:
It looks better, renamed.